app/jobs/dhis2/bangladesh_disaggregated_exporter_job.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Dhis2
+  class BangladeshDisaggregatedExporterJob < Dhis2ExporterJob
+    STEP = 5
+    BUCKETS = (15..75).step(STEP).to_a
```

Comment:
What's the relevance of 15 and 75?

Reply:
They're the age range across which the age buckets are split. I'll put them in constants as MIN_AGE and MAX_AGE for readability